DuckDuckGo/AppUserDefaults.swift

```diff
@@ -64,6 +67,30 @@ public class AppUserDefaults: AppSettings {
         
     }
     
+    var autoClearAction: Int {
```

Comment:
Why not just have this return the enum or an optional? 

Likewise for the `autoClearTiming` value